staging/src/k8s.io/cloud-provider/options/options.go

```diff
@@ -166,6 +172,12 @@ func (o *CloudControllerManagerOptions) ApplyTo(c *config.Config, userAgent stri
 	if err = o.ServiceController.ApplyTo(&c.ComponentConfig.ServiceController); err != nil {
 		return err
 	}
+	if err = o.Webhook.ApplyTo(&c.ComponentConfig.Webhook); err != nil {
+		return err
+	}
+	if err = o.WebhookServing.ApplyTo(&c.WebhookSecureServing); err != nil {
```

Comment:
can WebhookServing be nil? will this panic?

Reply:
Not if the consuming cloud-controller-manager is using the provided functions to initialize their options, like `NewCloudControllerManagerOptions`, but since they could be overriding this I'll add a nil check here.